zstacklib/zstacklib/utils/sanlock.py

```
@@ -11,6 +11,11 @@
 SECTOR_SIZE_512 = 512
 SECTOR_SIZE_4K = 8*512
 BIG_ALIGN_SIZE = 8*1024**2
+EIO = -5
+SANLK_AIO_TIMEOUT = -202
+
+def io_failed(rv):
+    return rv == EIO or rv == SANLK_AIO_TIMEOUT
```

Ruff (0.12.2)

Use `rv in (EIO, SANLK_AIO_TIMEOUT)` instead of multiple equality comparisons

Replace with `rv in (EIO, SANLK_AIO_TIMEOUT)`

(SIM109)